src/docs/arch-session.dox

```diff
@@ -0,0 +1,51 @@
+/*! @arch_page arch-session Session
+
+@section arch_session_def Definition
+After a @ref arch-connection has been established between the application and WiredTiger, the
+application can start sending requests to WiredTiger using a session. A session is internally
+represented by WT_SESSION and plays an important role since almost all operations require one.
+
+A session can only be created through an existing connection with the API
+WT_CONNECTION::open_session and it is possible to create multiple sessions through the same
+connection. In fact, one connection can have multiple sessions but one session can only be
+associated with one connection. The maximum number of sessions is set through the configuration item
+\c session_max as part of the configuration string in ::wiredtiger_open.
+
+Sessions that are created by the calling application are called "user sessions". WiredTiger also
+performs some internal operations such as @ref arch-eviction through self-created sessions. These
+sessions are called "internal sessions". The usage rules and guidelines for both internal sessions
+and user sessions are the same and the only difference between them is their origin of creation.
+
+@section arch_session_ops Operations
+The different operations that can be performed on a WiredTiger session are related to cursors
```

Comment:
cursors'

Reply:
What is cursors handles

Reply:
See the section dedicated to cursor handles : https://source.wiredtiger.com/develop/struct_w_t___s_e_s_s_i_o_n.html
The API is open_cursor

Reply:
Yes, you are talking about the schema operations. I wouldn't call then handles as it is confusing with dhandles.

Reply:
See 14e531c4083f3864a614c7812c433a8f5227914d